### ISSUE TITLE
Error when installing: no matching record

### ISSUE THREAD

**Issue by elektrinis** [alan196/crante#4]
Hello,

Tried to install this on Ubuntu Odoo 8.0 and get this error:

`Error while importing module 'visitor_log': u"Module loading visitor_log failed: file /tmp/tmp8H3FS2/visitor_log/security/ir.model.access.csv could not be processed:\n Line 1 : No matching record found for external id 'model_visitor_log' in field 'Object'"`

Is it a problem with server configuration?

**Comment by alan196** [comment]
No, this module is not installable for Odoo 8.0, it was developed in OpenERP 7.0 and then the customer abandon this project. 

This error is because the module refers to a XML ID that is not present in Odoo 8.0
